libs/blockscout-service-launcher/src/settings.rs

```diff
@@ -37,7 +37,7 @@ impl Default for GrpcServerSettings {
     fn default() -> Self {
         Self {
             enabled: true,
```

Comment:
may be set this to false for default?

Reply:
We may. Don't you think it will be too confusing?

Reply:
Okay, maybe not. I agree